test/Interop/Cxx/value-witness-table/Inputs/basic.h

```diff
@@ -0,0 +1,25 @@
+struct HasUserProvidedCopyConstructor {
+  int numCopies;
+  HasUserProvidedCopyConstructor(int numCopies = 0) : numCopies(numCopies) {}
+  HasUserProvidedCopyConstructor(const HasUserProvidedCopyConstructor &other)
+      : numCopies(other.numCopies + 1) {}
+};
```

Comment:
Consider renaming the file to reflect better its contents. `copy-constructors.h`